tbans/workers/subscription_worker.py

```diff
@@ -0,0 +1,72 @@
+import webapp2
+import logging
+
+
+class SubscriptionWorkerHandler(webapp2.RequestHandler):
+
+    @staticmethod
+    def task_queue():
+        from google.appengine.api import taskqueue
```

Comment:
This just in time importing is a different style than we've used elsewhere. The advantage is less overhead if we never call this method?

Reply:
Yeah - it probably matters less for things outside of the TBANSService class. The idea is that - classes that need to call out to TBANS shouldn't need to import anything else. So doing the inline imports makes this possible. I'm just doing the inline imports everywhere (even when it's not necessarily needed because we're not exposing those classes) so it's all similar.

For example, right now when we're spinning up the client API, we have to pull in GDCV, since we just do huge imports at the top of the file. You shouldn't *need* to pull in GDCV to hit the test endpoint in the client API, but you do.

Reply:
That makes sense. It'd be nice if Python did its own lazy importing to avoid this.